content/guides/getting-started/component-setup.md

```diff
@@ -27,14 +27,14 @@ cypress open
 ```
 
 The Launchpad's setup wizard will do the following things:
-
 1. Determine what changes need to be merged into your Cypress Config file.
+<!-- see GitHub comment-->
 2. Create a component support file for [configuring global styles]() and
```

Comment:
Empty link: `[configuring global styles]()`

Reply:
This guide should actually be removed, its content was migrated to Framework Configuration.

Reply:
which we'll take care of